spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -408,6 +408,8 @@ executing the `@LRA` annotated method, the implementation must be able to guaran
 participant has been or will eventually be registered with the LRA.
 This implies that the LRA is effectively validated as being active (since
 it is not possible to enlist with an inactive LRA).
+footnoteref:[note1,there is a caveat for nested LRAs which have an extra state model transition which allows an inactive LRA to transition from the `Closed` state to the `Cancelling` state. To facilitate this transition we allow enlistement if the implementation knows that the incoming context is nested and in the state `Closed` or `Active`. And in the case where the LRA is `Closed`, if the method finishes without error, the complete method, if present, would be invoked as normal.]
```

Comment:
or Status if Complete is not idempotent.

Reply:
This footnote is to cover the situation where a participant wishes to join with a closed nested LRA (but the parent LRA is still active) and therefore needs to be notified when the nested LRA closes.